jenkins/job-configs/kubernetes-jenkins/kubernetes-garbagecollector.yaml

```diff
@@ -72,5 +72,115 @@
                 export KUBEMARK_NUM_NODES="100"
                 # The kubemark scripts build a Docker image
                 export JENKINS_ENABLE_DOCKER_IN_DOCKER="y"
+        - '100-gce': # kubernetes-garbagecollector-100-gce
```

Comment:
But a successful 500-gce test doesn't ensure a 2000-gce will pass. If we want to avoid duplication, I think we can skip the 500-gce test. @gmarek what do you think?

Currently none of the scalability test is presubmit. Maybe we can promote the 100-gce one to be presubmit? I don't know if the result of 100-gce test is useful.